backend/src/graph_builder.py

```diff
@@ -10,17 +10,26 @@
 import langgraph.checkpoint.memory as lgcm
 
 
-TOOL_LIST_LOOKUP = {
+AGENT_LOOKUP = {
     "default": [
-        tools.get_action_items,
-        tools.parse_pdf,
-        tools.get_stock_price,
-        tools.parse_file,
-        tools.parse_webpage,
-        tools.summarize_text,
+        'Action Item Extractor',
+        'PDF Parser',
+        'Stock Price Fetcher',
+        'File Parser',
+        'Webpage Parser',
+        'Text Summarizer',
     ],
 }
 
+SKILL_LOOKUP = {
+    'Action Item Extractor': tools.get_action_items,
+    'PDF Parser': tools.parse_pdf,
+    'Stock Price Fetcher': tools.get_stock_price,
+    'File Parser': tools.parse_file,
+    'Webpage Parser': tools.parse_webpage,
+    'Text Summarizer': tools.summarize_text,
+}
+
 
 def tools_node(state: st.State, tools: list[callable]):
     tools_by_name = {tool.name: tool for tool in tools}
```

Comment:
that would be the `SKILL_LOOKUP` wouldn't it?

Reply:
Yes it is SKILL_LOOKUP